pkg/reconciler/serverlessservice/resources/services.go

```diff
@@ -98,15 +111,22 @@ func filterSubsetPorts(targetPort int32, subsets []corev1.EndpointSubset) []core
 	}
 	ret := make([]corev1.EndpointSubset, len(subsets))
 	for i, sss := range subsets {
-		sst := sss.DeepCopy()
```

Comment:
guess this DeepCopy() was not needed from the beginning because subsets were passed in by-value ?